decidim-dev/lib/decidim/test/rspec_support/capybara.rb

```diff
@@ -42,7 +42,7 @@ def switch_to_default_host
 }
 
 Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, capybara_options)
+  Capybara::Poltergeist::Driver.new(app, capybara_options.merge(screen_size: [1600, 1200]))
```

Comment:
Yeah, still it might be a good idea to use an actual resolution (mediaqueries could fail somewhere in the middle without it being a deal-breaker).